views/random-beer.hbs

```diff
@@ -0,0 +1,20 @@
+<div>
+
+    {{#each randomBeer}}
+        {{!-- <h1>{{name}}</h1>
+      <h3>{{tagline}}</h3>  
+         {{description}} --}}
+          {{> beerpartial this}}
+           <h3>Food Pairing</h3>
+        <ul>
+         {{#each this.food_pairing}}
```

Comment:
You can use "this" here if you'd like, but then I would be consistent and add it in front of "brewers_tips" too.